neptune/new/internal/utils/images.py

```diff
@@ -170,7 +171,7 @@ def _matplotlib_to_plotly(chart):
                 raise PlotlyIncompatibilityException(
                     matplotlib_version,
                     plotly_version,
-                    "Due to plotly using some deprecated matplotlib methods, we recommend downgrading matplotlib"
+                    "Due to Plotly using some deprecated Matplotlib methods, we recommend downgrading Matplotlib"
```

Comment:
@normandy7 I think it's more common to use lower letters only in names of python packages.

Reply:
Right, I guess these refer literally to the Python packages. Will revert